test/test_dynamic_shapes.py

```diff
@@ -195,13 +195,17 @@ def test_roundtrip(self):
 
         self.assertTrue(x.size()[0], 5)
         self.assertTrue(x.size()[1], 4)
-        self.assertTrue(isinstance(x.size()[1], int))  # due to guard above
+        # Should be simplifiable to an integer.
+        # Ref: https://github.com/pytorch/pytorch/pull/107492
+        self.assertTrue(isinstance(x.size()[1], SymInt))
```

Comment:
This seems worse; we specifically wanted specialized SymInt to decay to int as soon as possible?